### DISCUSSION TITLE
Expand Realtime Data

### DISCUSSION THREAD

**Discussion by bencreynolds** [pocketbase/pocketbase#505] [Q&A]
I’m using the JS SDK.

When I receive events, there is no way for me to see the expanded user profile. This ruins any use case I could think of for real-time updates, since you can literally only see the direct content and never any relational content. 

Is there a way to do this? There absolutely should be if not…

**Comment by ganigeorgiev** [comment]
Expand is intentionally disabled for the realtime events because it will require calling it for each subscribed client individually in order to verify that the receiver is allowed to view the expanded records and currently there is no optimal way to do it and will further increase the latency if allowed by default.

This may change in the future, but for now it is not possible to get the expanded relations together with the streamed event record.

As an alternative, you can treat the realtime event more like a notification/trigger and call on your own `client.records.getOne("example", "RECORD_ID", { "expand": "testRel" })` inside the subscribe callback (or directly request the records that you usually will expand).
